laravelServer/server/routes/console.php

```diff
@@ -17,3 +18,10 @@
 Artisan::command('inspire', function () {
     $this->comment(Inspiring::quote());
 })->purpose('Display an inspiring quote');
+Artisan::command('monthly-delete', function (){
+    $currentDay = date("Y-m-d");
+    $dateToDeleteFrom = date("Y-m-d", strtotime($currentDay."-1 Months"));
```

Comment:
change to preserve the last month and instead delete i-1 month. Sorry for vague explanation